src/eva/vision/utils/io/nifti.py

```diff
@@ -17,7 +17,7 @@ def read_nifti_slice(path: str, slice_index: int) -> npt.NDArray[Any]:
             return the full 3D image.
```

Comment:
`If `None`, it will return the full 3D image.`, is this still the case?

Reply:
yes

Reply:
Wouldn't this then raise an error?
`image_data.slicer[:, :, slice_index : slice_index + 1]`

Also the type hint `slice_index: int` would require an update then?

Reply:
oh yeah you are right, we have to remove this docs - basically the function reads only slices - updated!